.github/workflows/measure-packages-size.yml

```diff
@@ -0,0 +1,26 @@
+name: Measure packages size
+
+on:
+  workflow_dispatch:
+    inputs:
+      prNumber:
+        description: "PR Number"
+        required: true
+
+jobs:
+  measure-utils-sizes:
+    runs-on: ubuntu-latest
+    env:
+      NODE_ENV: dev
+      PR_NUMBER: ${{ inputs.prNumber }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Packages size report
+        uses: flochaz/pkg-size-action@v1.2.15
```

Comment:
TODO: update this version once https://github.com/flochaz/pkg-size-action/pull/1 is merged & a new version has been released